content/api/commands/origin.md

```diff
@@ -0,0 +1,467 @@
+---
+title: origin
+---
+
+Visit multiple domains of different
+[origin](https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy#definition_of_an_origin)
+in a single test.
+
+In normal use, a single Cypress test may only run commands in a single origin, a
+limitation determined by standard web security features of the browser. The
+`cy.origin()` command allows your tests to bypass this limitation.
+
+<Alert type="warning">
+
+<strong class="alert-header"><Icon name="exclamation-triangle"></Icon>
+Experimental</strong>
+
+TODO Maybe rework this & move to a partial
+
+The `origin` API is currently experimental, and can be enabled by setting
+the [`experimentalSessionAndOrigin`](/guides/references/experiments) flag
+to `true` in the Cypress config or by
+using [`Cypress.config()`](/api/cypress-api/config) at the top of a spec file.
```

Comment:
The new flag (experimentalSessionAndOrigin) is only configurable globally and can't be set in a spec file.